source/NVDAObjects/UIA/__init__.py

```diff
@@ -923,7 +923,11 @@ def setEndPoint(self,other,which):
 	def updateSelection(self):
 		self._rangeObj.Select()
 
-	updateCaret = updateSelection
+	def updateCaret(self) -> None:
+		copyTextInfo = self.copy()
+		copyTextInfo.collapse()

```

Comment:
Why does self have a range? (I.E. not collapsed already?)
Is it to keep track of the old vs new position?
Does this behavior match other `TextInfo` implementations?

Reply:
The behaviour I am introducing is the norm for other TextTnfos

https://github.com/nvaccess/nvda/blob/6f211a13fc1d9e2dc6400504173c314d917391a7/source/NVDAObjects/window/winword.py#L1052-L1056

https://github.com/nvaccess/nvda/blob/07add7e0862ff3340e49c426741f9010f1e341fe/source/NVDAObjects/window/edit.py#L790-L794

https://github.com/nvaccess/nvda/blob/b95743da959148924183effa98ffcc7f6e5f0349/source/cursorManager.py#L459-L463

After a non comprehensive search, I found one other instance which uses the current behaviour - MSHTML.
I have opted to fix this one as well to be consistent.
https://github.com/nvaccess/nvda/blob/17e52866e1dfc353edcc7dc9d466449e7920a155/source/NVDAObjects/IAccessible/MSHTML.py#L404-L408

Reply:
The range is intentionally a range selection as it being used in say all, which creates text infos in chunks to be spoken.